### PR TITLE
Add release notes in the repository

### PR DESCRIPTION
**Pull request by t-bast**
It's useful to have the release notes directly inside the repository instead of relying on Github: that lets for example users read them offline.

It also lets us fill the release notes every time we add relevant features on `master` instead of having to do them all at once when we decide to make a release. Every time a contributor makes a PR that should be included in the release notes, we should ask the contributor to update the release notes directly inside the PR. That ensures release notes are thorough and accurate, and splits the work between all contributors.

I copy-pasted the release notes of the last release (0.6.1) and create the draft release notes for the next release, so that we can start this process ASAP. I already filled the next release notes with everything noteworthy that happened since v0.6.1. @thomash-acinq can you please double-check what I put for A/B testing? And let me know if you want to add something about the changes you made to the heuristics (in particular the virtual hop cost).

### CODE REVIEW

File: docs/release-notes/eclair-vnext.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+In case the channel is unilaterally closed, eclair will use CPFP and RBF to ensure that transactions confirm in a timely manner.
+You **MUST** ensure you have some utxos available in your Bitcoin Core wallet for fee bumping, otherwise there is a risk that an attacker steals some of your funds.
+
+Do note that anchor outputs may still be unsafe in high-fee environments until the Bitcoin network provides support for [package relay](https://bitcoinops.org/en/topics/package-relay/).
+
+### Path-finding improvements
+
+This release contains many improvements to path-finding and paves the way for future experimentation.
+It adds support for A/B testing and various refinements of the previous heuristics.
+
```
Comment (thomash-acinq) [inline docs/release-notes/eclair-vnext.md:49]:
```suggestion
This release adds a new path-finding setting: a virtual cost per hop that can be used to penalize long paths.

It also adds support for A/B testing to find out which values work best.
```

File: docs/release-notes/eclair-vnext.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+```
+
+That should generate `eclair-node/target/eclair-node-<version>-XXXXXXX-bin.zip` with sha256 checksums that match the one we provide and sign in `SHA256SUMS.asc`
+
+(*) You may be able to build the exact same artefacts with other operating systems or versions of JDK 11, we have not tried everything.
+
+## Upgrading
+
+This release is fully compatible with previous eclair versions. You don't need to close your channels, just stop eclair, upgrade and restart.
+
```
[the remark this answers is not in the document]
Comment (t-bast) [reply docs/release-notes/eclair-vnext.md:168]: Good catch, this is an important caveat to mention!

File: docs/release-notes/eclair-vnext.md (hunk 1 above)
Comment (t-bast) [reply docs/release-notes/eclair-vnext.md]: Done in https://github.com/ACINQ/eclair/pull/1951/commits/26fe219e208a98326dfd260d58e24b70d1e5358c

File: docs/release-notes/eclair-vnext.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+### Tor support for blockchain watchdogs
+
+Eclair introduced blockchain watchdogs in v0.5.0, where secondary blockchain sources are regularly queried to detect whether your node is being eclipsed.
+
+Most of these watchdogs were previously queried over HTTPS, which exposes your IP address.
+This is fixed in this release: when using Tor, the watchdogs will now also be queried through Tor, keeping your IP address private.
+
+You can also now choose to disable some watchdogs by removing them from the `eclair.blockchain-watchdog.sources` list in `eclair.conf`.
+Head over to [reference.conf](https://github.com/ACINQ/eclair/blob/master/eclair-core/src/main/resources/reference.conf) for more details.
+
```
[the remark this answers is not in the document]
Comment (t-bast) [reply docs/release-notes/eclair-vnext.md:108]: Done in https://github.com/ACINQ/eclair/pull/1951/commits/26fe219e208a98326dfd260d58e24b70d1e5358c

File: docs/release-notes/eclair-vnext.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (t-bast) [reply docs/release-notes/eclair-vnext.md:181]: I added a warning about that to the `Per node relay fees` section in https://github.com/ACINQ/eclair/pull/1951/commits/26fe219e208a98326dfd260d58e24b70d1e5358c

**Review by thomash-acinq — DISMISSED** [review]

**Review by pm47 — COMMENTED** [review]
Should we add guidance on when/how to edit this file in the `CONTRIBUTING.md`?

**Comment by t-bast** [comment]
> Should we add guidance on when/how to edit this file in the CONTRIBUTING.md?

Good idea, done in https://github.com/ACINQ/eclair/pull/1951/commits/75ae8dd1dc5c6a33826d12a54970e4a30fb765b7

**Review by pm47 — APPROVED** [review]

**Pull request merged**